account_move_line_product/views/account_move_line_view.xml

```diff
@@ -3,20 +3,6 @@
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 
 <odoo>
-    <record id="view_move_line_form" model="ir.ui.view">
-        <field name="name">account.move.line.form</field>
-        <field name="model">account.move.line</field>
-        <field name="inherit_id" ref="account.view_move_line_form"/>
-        <field name="arch" type="xml">
-            <field name="quantity" position="before">
-                <field name="product_id"/>
-            </field>
-            <field name="quantity" position="attributes">
-                <attribute name="invisible">False</attribute>
-            </field>
-
-        </field>
-    </record>
```

Comment:
I remove this code because odoo v.13 showed `product_id` in this view.

![2019-12-17_13-46](https://user-images.githubusercontent.com/51266019/70971617-f93e0100-20d3-11ea-9340-89d6c89b664d.png)

Reply:
And is it annoying as the module is aimed for that purpose?